ledger/ledger-api-integration-tests/src/test/lib/scala/com/digitalasset/platform/apitesting/LedgerContext.scala

```diff
@@ -75,6 +76,38 @@ trait LedgerContext {
     * */
   def reset()(implicit system: ActorSystem, mat: Materializer): Future[LedgerContext]
 
+  /**
+    *  Return a copy of the current ledger context, where all gRPC calls use the given call credentials.
+    *  Use for tests that need to override call credentials on a per-test basis.
+    *  @return the new LedgerContext
+    */
+  def withCallCredentials(credentials: CallCredentials): LedgerContext
+
+  /**
+    *  Return a copy of the current ledger context, where all gRPC calls use the given Authorization header.
+    *  Use for tests that need to override call credentials on a per-test basis.
+    *  @return the new LedgerContext
+    */
+  def withAuthorizationHeader(headerValue: String): LedgerContext = {
+    val auth = Metadata.Key.of("Authorization", Metadata.ASCII_STRING_MARSHALLER)
+
+    val callCredentials = new CallCredentials {
+      override def applyRequestMetadata(
+          requestInfo: CallCredentials.RequestInfo,
+          appExecutor: Executor,
+          applier: CallCredentials.MetadataApplier): Unit = {
+        val metadata = new Metadata
+        metadata.put(auth, headerValue)
```

Comment:
```suggestion
        metadata.put(auth, s"Bearer $headerValue")
```
Should it not be like this?